components/dependency_injection/autowiring.rst

```diff
@@ -308,7 +308,10 @@ and a Twitter client using it:
         services:
             rot13_transformer:
                 class:            Acme\Rot13Transformer
-                autowiring_types: Acme\TransformerInterface
+
+            Acme\TransformerInterface:
+                alias: rot13_transformer
+                public: false
```

Comment:
I don't think we should bother with this kind of optimization that can lead to confusion here. And as it's not specific to autowiring, I should rather be in a dedicated section rather. The notion of public/private is a bit special in Symfony, I wouldn't mix it with autowiring.